### ISSUE TITLE
Project status

### ISSUE THREAD

**Issue by ansarizafar** [redwood/redwood#165]
Redwood is an amazing project and It can be game changer. Is it possible to use it for a traditional web application today? Can we compare Redwood with CondensationDB https://condensationdb.com/

**Comment by ansarizafar** [comment]
@brynbellomy  I am still waiting for reply.

**Comment by brynbellomy** [comment]
Yes, you can build a web application on it today. We have proven that use case a few times already.

I'd love to have a conversation, but your question is very open-ended. Which aspects of CondensationDB do you want to compare against Redwood? I've reviewed the Condensation docs to some extent, and it seems like you guys have some good ideas. But let's focus the discussion on some specifics that are most interesting to you. Or, perhaps, you can come to a Braid meeting. https://braid.org/ (https://braid.org/meeting-23 should appear at some point in the coming week or so)
